<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1382</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>NanumGothic</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background: white;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="font">
       <font>
        <family>NanumGothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <widget class="QTableWidget" name="tableWgt1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1200</width>
         <height>580</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#tableWgt1 {
	border: 2px solid rgb(203, 235, 255);
}

QTableWidget QHeaderView {
	background: rgb(250, 255, 255);
}</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>호스트명</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CPU</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>메모리</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>메모리 증감율</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>디스크</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>디스크 용량</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>디스크 증감율</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>서비스</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>작업스케줄러</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>윈도우디펜더</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>이벤트</string>
        </property>
        <property name="font">
         <font>
          <family>NanumGothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
      </widget>
      <widget class="QToolButton" name="toolBtn3">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>500</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>NanumGothic</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border: 2px solid rgb(203, 235, 255);
	background: rgb(203, 235, 255);
}</string>
       </property>
       <property name="text">
        <string>서버 삭제</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolBtn1">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>0</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>NanumGothic</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border: 2px solid rgb(203, 235, 255);
	background: rgb(203, 235, 255);
}</string>
       </property>
       <property name="text">
        <string>새로고침</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolBtn2">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>460</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>NanumGothic</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border: 2px solid rgb(203, 235, 255);
	background: rgb(203, 235, 255);
}</string>
       </property>
       <property name="text">
        <string>서버 추가</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolBtn4">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>290</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>NanumGothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#toolBtn4 {
	border: 2px solid rgb(235, 203, 255);
	background: rgb(235, 203, 255);
}</string>
       </property>
       <property name="text">
        <string>지난 날짜 불러오기</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit1">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>260</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="toolBtn5">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>540</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>NanumGothic</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border: 2px solid rgb(203, 235, 255);
	background: rgb(203, 235, 255);
}</string>
       </property>
       <property name="text">
        <string>코드 관리</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
